.buildkite/scripts/build/pre-build-operator.sh

```diff
@@ -9,7 +9,7 @@
 
 set -euo pipefail
 
-ROOT="$(cd "$(dirname "$0")"; pwd)/../../.."
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)/../../.."
```

Comment:
This is required because the script is now "sourced", therefore `dirname "$0"` is the directory from which `pre-build-operator.sh` is sourced, not the directory of the script itself.